openquake/risklib/riskinput.py

```diff
@@ -116,17 +116,20 @@ def __len__(self):
 
     def __toh5__(self):
         attrs = {'time_event': self.time_event or 'None',
-                 'time_events': self.time_events,
-                 'loss_types': hdf5.array_of_vstr(self.loss_types),
-                 'deduc': hdf5.array_of_vstr(self.deduc),
-                 'i_lim': hdf5.array_of_vstr(self.i_lim),
-                 'retro': hdf5.array_of_vstr(self.retro),
+                 'time_events': ' '.join(map(decode, self.time_events)),
```

Comment:
Please add a comment about the fact none of the joined items can ever contain spaces